package-lock.json

```diff
@@ -4,7 +4,6 @@
     "requires": true,
     "packages": {
         "": {
-            "name": "thingweb.node-wot",
```

Comment:
Minor problem, when I run `npm install` this line is added automatically by my npm, can we re-introduce it? Otherwise it will pop up again later in some other PR 😄

Reply:
Sure! :) I am a bit clueless why my npm messes up the `package-lock.json`, though :/

Reply:
I added it manually now – maybe you could update the `package-lock.json` file on the master branch once the PR is merged just to be safe? :)